daliuge-common/docker/Dockerfile.devcuda

```diff
@@ -7,23 +7,18 @@ FROM ubuntu:20.04
 ARG BUILD_ID
 LABEL stage=builder
 LABEL build=$BUILD_ID
-RUN apt-get update && apt-get install -y gcc python3 python3.8-venv && apt-get clean
+RUN apt-get update && \
+    apt-get install -y gcc python3 python3.8-venv python3-pip python3-distutils libmetis-dev curl git sudo && \
+    apt-get clean
 
 COPY / /daliuge
 
-RUN cd && python3 -m venv dlg && cd /daliuge && \
-    . ${HOME}/dlg/bin/activate && \
-    pip install numpy && \
-    pip install . && \
-    apt-get remove -y gcc && \
-    apt-get autoremove -y
+RUN cd / && python3 -m venv dlg && cd /daliuge && \
+    . /dlg/bin/activate && \
+    pip install wheel numpy && \
+    pip install .
 
-
-FROM ubuntu:20.04
-RUN apt-get update && apt-get install -y bash
-COPY --from=0 /root/dlg /root/dlg
-
-RUN apt install -y wget gnupg2 software-properties-common
+RUN DEBIAN_FRONTEND=noninteractive apt install -y wget gnupg2 software-properties-common
```

Comment:
Updated to match Dockerfile.dev. Basically cuda support is an extra docker build stage between common and engine, but there isn't a docker command to include a seperate dockerfile